docs/develop/host-specific-api-model.md

```diff
@@ -125,17 +157,17 @@ someRange.load("format/font/name")
 
 You can also set the scalar properties of a navigation property by traversing the path. For example, you could set the font size for an `Excel.Range` by using `someRange.format.font.size = 10;`. You don't need to load the property before you set it.
 
-#### Calling `load` without parameters
+Please be aware that some of the properties under an object may have the same name as another object. For example, `format` is a property under `Exce;.Range` object, but `format` itself is an object as well. So, if you make a call such as `range.load("format")`, this is equivalent to `range.format.load()`, which is an empty `load()` call that can cause performance problems as outlined previously. To avoid this, your code should only load the "leaf nodes" in an object tree.
```

Comment:
```suggestion
Please be aware that some of the properties under an object may have the same name as another object. For example, `format` is a property under the `Excel.Range` object, but `format` itself is an object as well. So, if you make a call such as `range.load("format")`, this is equivalent to `range.format.load()`, which is an empty `load()` call that can cause performance problems as outlined previously. To avoid this, your code should only load the "leaf nodes" in an object tree.
```

Reply:
Something feels off to me about the 'as outlined previously' phrase but I hesitate to remove it. Would the paragraph be clear without it?

Reply:
How about:
"Please be aware that some of the properties under an object may have the same name as another object. For example, `format` is a property under the `Excel.Range` object, but `format` itself is an object as well. So, if you make a call such as `range.load("format")`, this is equivalent to `range.format.load()` (an undesirable empty `load()` statement). To avoid this, your code should only load the "leaf nodes" in an object tree."

Reply:
Yes, looks good to me! More concise.